requirements.txt

```diff
@@ -3,4 +3,4 @@ pulpcore>=3.20.0.dev
 ecdsa~=0.14
 pyjwkest~=1.4.0
 pyjwt[crypto]~=1.7.1
-jsonschema~=4.4.0
+jsonschema>=4.4
```

Comment:
@mdellweg what should we use? ~=4.4 or >=4.4, <4.6?
The moment ansible-list will change their deps to 4.6.z - we will have an issue
The moment jsonschema introduces breaking changing in 4.x - we will have an issue

Same approach should take pulp-ansible too.

Reply:
Yeah, it's a tough call. On the one hand, once we need to introduce a stricter dependency in the future (for such a breaking change), pip will come back trying to install this less strict version of pulp_container, leading to a broken system.
OTOH, can we expect jsonschema to break ever? And at least they claim semantic versioning: https://python-jsonschema.readthedocs.io/en/latest/faq/#how-do-jsonschema-version-numbers-work .

Reply:
if they follow sem ver then it's ok to have ~=4.4